docs/en/getting_started.md

```diff
@@ -192,6 +192,15 @@ you can install it before installing MMCV.
 
 4. Some dependencies are optional. Simply running `pip install -v -e .` will only install the minimum runtime requirements. To use optional dependencies like `albumentations` and `imagecorruptions` either install them manually with `pip install -r requirements/optional.txt` or specify desired extras when calling `pip` (e.g. `pip install -v -e .[optional]`). Valid keys for the extras field are: `all`, `tests`, `build`, and `optional`.
 
+   We have supported spconv2.0. If you have installed spconv2.0, the code will use spconv2.0 first, and it has better performance. If necessary please refer to [spconv v2.x](https://github.com/traveller59/spconv) or use pip:
```

Comment:
We have supported spconv2.0. If you have installed spconv2.0, the code will use spconv2.0 first, which could reduce the GPU memory. Users can use the following commands to install spconv2.0,
```bash
pip install cumm-cuxxx
pip install spconv-cuxxx
```
where `xxx` is the CUDA version in the environment.
For example, if you are using CUDA 10.2, the command will be `pip install cumm-cu102
 && pip install spconv-cu102`.
Supported CUDA versions include 10.2, 11.1, 11.3, 11.4. Users can also install it by building from source.
For more details please refer to [spconv v2.x](https://github.com/traveller59/spconv).